AdminGuide.md

```diff
@@ -0,0 +1,156 @@
+# Admin Guide for WikiEduDashboardTools
+
+This guide provides detailed instructions for setting up, managing, and troubleshooting WikiEduDashboardTools on Toolforge and Cloud VPS.
+
+## Table of Contents
+1. [Setup Instructions](#setup-instructions)
+   - [Toolforge Setup](#toolforge-setup)
+   - [Cloud VPS Setup](#cloud-vps-setup)
+2. [Managing the System](#managing-the-system)
+   - [Toolforge](#toolforge)
+        - [Starting, Stopping, and Restarting Services](#starting-stopping-and-restarting-services)
+        - [Monitoring and Logs](#monitoring-and-logs)
+        - [Some useful toolforge commands](#some-useful-toolforge-commands)
+   - [Cloud VPS](#cloud-vps)
+        - [Starting, Stopping, and Restarting Instances](#starting-stopping-and-restarting-instances)
+        - [Monitoring](#monitoring)
+        - [Some useful commands](#some-useful-commands)
+3. [Troubleshooting](#troubleshooting)
+    - [Web Server Issues](#web-server-issues)
+    - [Database Issues](#database-issues)
+    - [Data Dumps and Recovery](#data-dumps-and-recovery)
+4. [More Resources](#more-resources)
+
+
+## Setup Instructions
```

Comment:
I don't think that setup instructions belong in this guide, as if services need to be set up from scratch, there's probably a reason that requires changes in the instructions.

I think the first thing this guide should have is an overview of all the system components and their roles, and the names of the servers in wmcloud that run each of them. Since the most common fix for a problem is to restart a server, and the next step is probably logging into that server to poke around and diagnose an unknown problem, that's main essential element of a guide in my opinion.